src/compiler/config/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const DEFAULT_DEV_MODE = false;
+export const DEFAULT_HASHED_FILENAME_LENTH = 8;
+export const MIN_HASHED_FILENAME_LENTH = 4;
+export const MAX_HASHED_FILENAME_LENTH = 32;
+export const DEFAULT_NAMESPACE = 'App';
+export const DEFAULT_FS_NAMESPACE = 'app';
```

Comment:
Some of these are copied out of `validate-config.ts`, and others (`DEFAULT_NAMESPACE`, `DEFAULT_FS_NAMESPACE`) are what I believe to be good defaults

Reply:
Well, if you made-up the namespace constants you did a good job cuz we have them hardcoded in [validate-namespace.ts](https://github.com/ionic-team/stencil/blob/main/src/compiler/config/validate-namespace.ts#L6)